source/common/CMakeLists.txt

```diff
@@ -20,12 +32,11 @@ target_include_directories(ur_common PUBLIC
 
 message(STATUS "Download Unified Memory Framework from github.com")
 if (NOT DEFINED UMF_REPO)
-    set(UMF_REPO "https://github.com/oneapi-src/unified-memory-framework.git")
+    set(UMF_REPO "https://github.com/igchor/unified-memory-framework.git")
```

Comment:
we merged https://github.com/oneapi-src/unified-memory-framework/pull/684 into main, but I guess we could stick with UMF commits from v0.9.x stable branch for now...?

I guess, you could only set UMF_TAG to main branch for L0_V2 adapter...? I'm not sure if this makes sense

Reply:
Hm, why do we actually use v0.9.x on main? We should definitely use v0.9.x for UR 0.10.0 but for UR main we can use UMF main I think

Reply:
och, you're right, we already have v0.10.x stable branch in UR